lib/plug/logger.ex

```diff
@@ -46,5 +46,6 @@ defmodule Plug.Logger do
   defp formatted_diff(diff), do: [Integer.to_string(diff), "µs"]
 
   defp connection_type(%{state: :chunked}), do: "Chunked"
```

Comment:
I thought that it would be used to log subsequent chunks as a response is streamed, but since `chunk/2` doesn't call the before_send callbacks, they won't be logged anyway.

I guess if you want to log streamed data in a long-running connection you have to do it yourself?